content/hardware/02.hero/boards/mega-2560/features.md

```diff
@@ -1,32 +1,27 @@
 
 <FeatureDescription>
+
 The Arduino Mega 2560 is a microcontroller board based on the ATmega2560. It has 54 digital input/output pins (of which 15 can be used as PWM outputs), 16 analog inputs, 4 UARTs (hardware serial ports), a 16 MHz crystal oscillator, a USB connection, a power jack, an ICSP header, and a reset button.
+
 </FeatureDescription>
 
 <FeatureList>
 
-<Feature title="ATmega2560" image="core">
-
-The high Performance, low power AVR® 8-bit microcontroller.
-
-<FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/ATmega640-1280-1281-2560-2561-Datasheet-DS40002211A.pdf" download blank/>
-</Feature>
-
 <Feature title="EEPROM" image="mcu">
```

Comment:
Can you add a link here that goes to:
- https://docs.arduino.cc/learn/programming/eeprom-guide